sdk/appconfiguration/azure-data-appconfiguration-v2/src/main/java/com/azure/v2/data/appconfiguration/implementation/SyncTokenPolicy.java

```diff
@@ -61,8 +61,7 @@ public void updateSyncToken(String token) {
             final SyncToken syncToken;
             try {
                 syncToken = SyncToken.createSyncToken(syncTokenString);
-            } catch (Exception ex) {
-                logger.logThrowableAsWarning(ex);
+            } catch (RuntimeException ex) {
```

Comment:
exceptions are logged in `SyncToken.createSyncToken`

Reply:
In that case, do we need a try/catch here?

Reply:
yes, we continue on the next line and swallow the exception